tooling/profiler/src/cli/execution_flamegraph_cmd.rs

```diff
@@ -174,6 +174,7 @@ fn ensure_brillig_entry_point(artifact: &ProgramArtifact) -> Result<(), CliError
     let err_msg = "Command only supports fully unconstrained Noir programs e.g. `unconstrained fn main() { .. }";
     let program = &artifact.bytecode;
     if program.functions.len() != 1 || program.unconstrained_functions.len() != 1 {
+        dbg!("got here");
```

Comment:
```suggestion
```

Reply:
I think you reviewed before [6f00ae0](https://github.com/noir-lang/noir/pull/7578/commits/6f00ae0ef16efd4970f7c4e260f8764dd376889d), removed this debug in that commit.